### ISSUE TITLE
Restore Does not work

### ISSUE THREAD

**Issue by AndrewKillian** [FABtotum/fabui-colibri#171]
When trying to update to 1.0 the machine gets stuck on the yellow lights, however strangely FABui can still be accessed with the yellow lights flashing. I attempted to use recovery mode saving no data but this had no effect so I tired to keep only the user settings, however this would not allow me to finish the process. There are two PHP errors at the top of the screen (array_merge(): Argument #2 is not an array and Invalid argument supplied for foreach() ) I go through the steps but I get stuck after #3 when the next button does not do anything I think I will try a fresh system next and will update with the results of that.

**Comment by AndrewKillian** [comment]
So a fresh boot worked and got the yellow lights to turn off but it would be nice to have kept all my settings, will try to update to 1.0 again now.

**Comment by AndrewKillian** [comment]
deleting everything did work and I was able to update
